swatch-core/src/main/resources/swatch-core/application.yaml

```
@@ -125,3 +125,4 @@ rhsm-subscriptions:
       kafka-group-id: offering-worker
   clowder:
     json-resource-location: file:/cdapp/cdappconfig.json
+    strictLoading: true
```

See other comment about lower kebab case...

```suggestion
    strict-loading: true
```